packages/quantic/force-app/main/default/lwc/quanticResultList/quanticResultList.js

```diff
@@ -73,17 +90,25 @@ export default class QuanticResultList extends LightningElement {
 
   disconnectedCallback() {
     this.unsubscribe?.();
+    this.unsubscribeSearchStatus?.();
+    this.unsubscribeResultsPerPage?.();
   }
 
   updateState() {
-    this.state = this.resultList.state;
+    this.state = this.resultList?.state;
+    this.numberOfResults = this.resultsPerPage?.state?.numberOfResults;
+    this.showPlaceholder = !this.searchStatus?.state?.firstSearchExecuted && !!this.numberOfResults;
```

Comment:
We might want to check `searchStatus.state.isLoading` as well to be sure components are ready to be displayed.